Project/src/MakeCall/CallCard.js

```diff
@@ -62,6 +64,7 @@ export default class CallCard extends React.Component {
             canShareScreen: this.capabilities.shareScreen.isPresent || this.capabilities.shareScreen.reason === 'FeatureNotSupported',
             canRaiseHands: this.capabilities.raiseHand.isPresent || this.capabilities.raiseHand.reason === 'FeatureNotSupported',
             canSpotlight: this.capabilities.spotlightParticipant.isPresent || this.capabilities.spotlightParticipant.reason === 'FeatureNotSupported',
+            canReact: this.capabilities.reaction.isPresent,
```

Comment:
Add the || condition for FeatureNotSupported use case to start with true for canReact say in case of GroupCall or 1:1 call.

Reply:
If FeatureNotSupported, why would I want my capability value to be true? 
It should be false as expected and the value is read from `isPresent` field. 

For group calls, we are not depending on the Capability API so we will send the reacitons.

Reply:
if we start with false, then the UI components will be disabled. so, add the  FeatureNotSupported condition now, to start with true.
Note: Will be updating the capabilities feature to throw exception in case of not supported call type, so we don't have to do this. 
For now, have both the conditions to start with true and not impact the UI for the group call and 1:1 call use case.